modules/tinymce/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Changed
+- `@toolbar-separator-color` variable is now exclusively used to determine the color of separator between Menubar and Toolbar instead of combination of `@menubar-row-separator-color` and `@toolbar-separator-color` #TINY-8632
```

Comment:
This doesn't look accurate, as the `@menubar-row-separator-color` is still used it's just transparent? Additionally the root change more so looks to be that there is no separator between the menubar and toolbar anymore.

Reply:
Not quite. `@menubar-row-separator-color` is now only applies to the divider between menubar rows (in case there is any menu items wrapping). The separator between menubar and toolbar is gone since v6 (visually) but can be added using `@toolbar-separator-color` variable. I tried to reword changelog entry to make it more clear, hope it helps